Makefile.core.mk

```diff
@@ -403,6 +403,10 @@ endif
 operator-name:
 	sed -i "s/\(projectName:\).*/\1 ${OPERATOR_NAME}/g" PROJECT
 
+.PHONY: operator-chart
+operator-chart:
+	sed -i -e "s/\(image: .*:\).*/\1${TAG}/" chart/values.yaml
```

Comment:
Would it be safer to overwrite whole IMAGE and not just TAG? This will not work if HUB, IMAGE_BASE are changed.

Reply:
good point. done!